src/api/volunteerRequest.ts

```diff
@@ -26,14 +28,51 @@ const router = Router();
  *                 items:
  *                   $ref: '#/components/schemas/volunteerRequest'
  */
-router.get('/', async (req: Request, res: Response, next: NextFunction) => {
+router.get('/volunteer', async (req: Request, res: Response, next: NextFunction) => {
   try {
     res.json(await getRelevantAndOpenVolunteerRequests());
   } catch (e) {
     next(e);
   }
 });
 
+/**
+ * @openapi
+ * paths:
+ *   /api/v1/volunteer-requests/manager:
```

Comment:
the route is problematic, because `volunteer-requests/{volunteerRequestId}` is saved for a specific volunteer request.
i think a better fit would be `/api/v1/prgoram/{programId}/volunteer-requests` since we want all of the requests of a certain program